chain/chain/src/store.rs

```diff
@@ -1860,39 +1860,29 @@ impl<'a> ChainStoreUpdate<'a> {
                 debug_assert_eq!(chunk.header.inner.height_created, height);
                 // 1a. Delete from receipt_id_to_shard_id (ColReceiptIdToShardId)
```

Comment:
All the comments like "// 1a. blah-blah" becomes unnecessary after adding explicit `gc_col`.